### PR TITLE
Make sure to not crash if receiving a pointer event for a disposed UIElement

### PR DESCRIPTION
**Pull request by dr1rrb**
GitHub Issue: 
https://github.com/unoplatform/uno/issues/2656

## Stability enhancement
Make sure to not crash if receiving a pointer event for a disposed `UIElement`

## What is the current behavior?
If a `UIElement` is explicitly disposed by application code, we might receive pointer events on non-initialized element, which causes `NullRefException`

## What is the new behavior?
We silently dismiss all native event received on non initialized `UIElement`

## PR Checklist
- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/UIElement.Pointers.Android.cs [hunk 1]
```diff
@@ -54,12 +54,9 @@ public bool OnTouch(View view, MotionEvent nativeEvent)
 
 		protected sealed override bool OnNativeMotionEvent(MotionEvent nativeEvent, View originalSource, bool isInView)
 		{
-			if (nativeEvent.PointerCount > 1
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/UIElement.Pointers.Android.cs]: Why is this removed ?

Comment (dr1rrb) [reply src/Uno.UI/UI/Xaml/UIElement.Pointers.Android.cs]: Because now we support multi touch :)

**Comment by jeromelaban** [comment]
@mergifyio backport release/beta/2.1
